### DISCUSSION TITLE
The meaning of solver parameter 'l-tol'

### DISCUSSION THREAD

**Discussion by echo1115** [idaholab/moose#26883] [Q&A General] [answered]
Hi all,

I want to know the definition of parameter `l-tol` in `[Executioner]`.  
For example, when solving $Ax=b$, then `l-tol` = ${||b-Ax||} \over {||b-Ax_0||}$ ?, or `l-tol` = ${||b-Ax||} \over {||b||}$ ?

Thanks in advance!

**Comment by GiudGiud** [comment, marked as answer]
Hello

l_tol is relative.
This question has a lot of nuances. You should complement this by looking the PETSc documentation for the solve_type of interest

Guillaume

**Reply by echo1115** [reply]
Thanks for your suggestion! I have got the answer I want in the[ PETSc documentation-Krylov Methods-Convergence Tests](https://petsc.org/release/manual/ksp/#convergence-tests) .

![image](https://github.com/idaholab/moose/assets/69078335/646f6697-f920-46d4-957a-163ef48b0220)
